webapp/platform/types/src/insights.ts

```diff
@@ -69,6 +69,7 @@ export type TopChannelActionResult = {
     error?: any;
 };
 
+// LOL
```

Comment:
LOL? Shouldn't this whole file be deleted?

Reply:
Not sure what this is about - maybe he left that as a reminder to delete? I noticed the same here: https://github.com/mattermost/mattermost/pull/23952#discussion_r1255016576

Reply:
Hey yes, LOL is like my personal TODO, sorry for missing its removal before pushing.